quickstart/README.md

```diff
@@ -0,0 +1,77 @@
+# Minikube quickstart
+
+### Install Minikube
+Follow instructions in https://minikube.sigs.k8s.io/docs/start/ to install the minikube.
+
+### Start Minikube
+```
+minikube start
+```
+
+### Enable Minikube `ingress` addon
+```
+minikube addons enable ingress
+```
+
+### Apply quickstart `.yml` configuration
+Assuming you cloned Lighter repo to `~/git/lighter`:
+1. Create main resources
+```
+minikube kubectl -- apply -f git/lighter/quickstart/lighter.yml
```

Comment:
maybe remove `git/lighter` prefix from the command? In that case user will be able to `cd  whatever_dir/lighter` and just copy/paste these commands

`minikube kubectl -- apply -f quickstart/lighter.yml`